novawallet/Common/Services/ExtrinsicService/Substrate/FeeManaging/ExtrinsicFeeEstimationRegistry.swift

```diff
@@ -110,20 +95,40 @@ extension ExtrinsicFeeEstimationRegistry: ExtrinsicFeeEstimationRegistring {
                 throw ExtrinsicFeeEstimationRegistryError.unexpectedChainAssetId(chainAssetId)
             }
 
-            switch AssetType(rawType: asset.type) {
-            case .none:
-                return CompoundOperationWrapper.createWithResult(ExtrinsicNativeFeeInstaller())
-            case .statemine:
-                let assetConversionInstaller = ExtrinsicAssetConversionFeeInstaller(
-                    feeAsset: ChainAsset(chain: chain, asset: asset)
-                )
-
-                return CompoundOperationWrapper.createWithResult(assetConversionInstaller)
-            case .orml, .equilibrium, .evmNative, .evmAsset:
-                throw ExtrinsicFeeEstimationRegistryError.unexpectedChainAssetId(chainAssetId)
-            }
+            return createFeeInstallerWrapper(
+                chain: chain,
+                asset: asset,
+                chainAssetId: chainAssetId
+            )
         } catch {
             return CompoundOperationWrapper.createWithError(error)
         }
     }
+
+    func createFeeInstallerWrapper(
+        chain: ChainModel,
+        asset: AssetModel,
+        chainAssetId: ChainAssetId
+    ) -> CompoundOperationWrapper<ExtrinsicFeeInstalling> {
+        switch AssetType(rawType: asset.type) {
+        case .statemine where chain.hasAssetHubTransferFees:
+            CompoundOperationWrapper.createWithResult(
+                ExtrinsicAssetConversionFeeInstaller(
+                    feeAsset: ChainAsset(chain: chain, asset: asset)
+                )
+            )
+        case .none, .statemine:
```

Comment:
If we don't have a flag for ```statemine``` asset then it means asset is not supported for fees